src/electron/services/heicConverter.ts

```
@@ -0,0 +1,185 @@
+      const files = await fs.readdir(this.cacheDir);
+      const cutoffTime = Date.now() - 7 * 24 * 60 * 60 * 1000; // 7 days ago
+
+      for (const file of files) {
+        try {
+          const filePath = path.join(this.cacheDir, file);
+          const stats = await fs.stat(filePath);
+
+          if (stats.mtime.getTime() < cutoffTime) {
+            await fs.unlink(filePath);
+          }
+        } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)